### PR TITLE
Release Notes for October 2022

### PR DESCRIPTION
**Pull request by can-memfault**
I didn't forget the truncate this month! :D

### CODE REVIEW

File: blog/2022-10-31.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: September 2022
```
Comment (can-memfault) [inline blog/2022-10-31.mdx:2]: Oops, the title

File: blog/2022-10-31.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+[code samples](/docs/best_practices/metrics-for-battery-life/#implementing-battery-metrics-in-code)
+help with understanding battery life of MCU, Linux or Android devices.
+
+<!-- truncate -->
+
+## General
+
+- Added:
+  [Documentation of best practices for battery life tracking](/docs/best_practices/metrics-for-battery-life)
+- Added: Timezone picker for selecting custom timezones
```
Comment (HBehrens) [inline blog/2022-10-31.mdx:26]: BTW, I feel this could have been another highlight… too late (and not critical) now

File: blog/2022-10-31.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    to build and run correctly with
+    [nRF-Connect SDK v2.1.0](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-2.1.0.html)
+  - Fixed: Compilation for the
+    [ESP32-S3](https://www.espressif.com/en/products/socs/esp32-s3)
+
+## Android
+
+- Added: Creation of kernel oops traces from `last_kmsg`
+- Improved: OTA link expiry window increased to 6hr (previously 1hr) for more
+  reliable downloads on slow internet speeds
```
Comment (HBehrens) [inline blog/2022-10-31.mdx:60]:
```suggestion
  reliable downloads on slow internet connections
```

Sorry, I know I wrote that myself but "slow speed" should rather be "low speed" right? As we're changing this, I think "slow connection" sounds more technical anyway

**Review by HBehrens — APPROVED** [review]
One minor suggestion on wording. If you apply that, please make sure to back-port that to Notion so that Marketing can copy&paste from there

Otherwise LGTM!

**Pull request merged**